atc/api/ccserver/get.go

```diff
@@ -1,14 +1,14 @@
 package ccserver
 
 import (
+	"code.cloudfoundry.org/lager/v3"
 	"encoding/xml"
 	"fmt"
+	"github.com/concourse/concourse/atc/api/accessor"
 	"net/http"
 	"net/url"
 	"time"
 
-	"code.cloudfoundry.org/lager/v3"
-
 	"github.com/concourse/concourse/atc"
 	"github.com/concourse/concourse/atc/db"
 	"github.com/tedsuo/rata"
```

Comment:
nit: the import are out of order. run `go fmt` on this file and any of the other files you've modified to fix this automatically for you.